cmake/RINGMesh.cmake

```diff
@@ -153,6 +153,7 @@ install(
 
 # Exports RINGMesh target
 include(CMakePackageConfigHelpers)
+set(CMAKE_INSTALL_SYSTEM_RUNTIME_DESTINATION bin/$<CONFIGURATION>)

```

Comment:
This will not work on Unix system use ${CMAKE_BUILD_TYPE}.

Reply:
I thought that this path was for window